mantidproject/mantid
# Muon double pulse analysis

**Description of work.**

Some Muon data comes from double pulse experiments. Ideally these need to be analysed by fitting two functions corresponding to the two pulses.

This adds double pulse analysis to the Muon analysis GUI. See the updated documentation for details of implementation. 



**To test:**
  * Scientists have said they will test whether this works in practise once the changes are in nightly.
  * In Muon analysis try doing some fits both with and without the double pulse option ticked and check everything is stable and the fits make sense.



Fixes #28741  




---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
